website/config/routes.js

```diff
@@ -596,6 +596,7 @@ module.exports.routes = {
   'GET /learn-more-about/policy-automation-run-script': '/guides/policy-automation-run-script',
   'GET /learn-more-about/installing-fleetctl': '/guides/fleetctl#installing-fleetctl',
   'GET /learn-more-about/mdm-disk-encryption': '/guides/enforce-disk-encryption',
+  'GET/learn-more-about/encrypt-linux-device': '/guides/linux-disk-encryption-end-user',
```

Comment:
@eashaw how do guide URLs work? Does the naming follow the file name or the title?